### PR TITLE
Feature/snek 25 difficulty

### PR DESCRIPTION
**Pull request by benniii1**

### CODE REVIEW

File: src/main/java/at/ac/fhcampuswien/snake/board/GameBoard.java [hunk 1]
```diff
@@ -162,7 +164,6 @@ private void initializeBoardObjects() {
         innerWall = generateRandomWall();
         food = new Food(snake, innerWall);
 
-        drawGameBoard(gc);
```
Comment (sorinlzr) [inline src/main/java/at/ac/fhcampuswien/snake/board/GameBoard.java:165]: Why did you remove this?

File: src/main/java/at/ac/fhcampuswien/snake/controller/MainViewController.java [hunk 2]
```diff
@@ -54,6 +58,12 @@ public void initialize() {
      */
     @FXML
     public void startGame() throws IOException {
+        String difficulty = comboBox.getValue().toString();
+        switch (difficulty) {
+            case "Easy" -> StateManager.refreshTime = 300;
+            case "Medium" -> StateManager.refreshTime = 200;
+            case "Hard" -> StateManager.refreshTime = 150;
+        }
```
Comment (sorinlzr) [inline src/main/java/at/ac/fhcampuswien/snake/controller/MainViewController.java:66]: I would put these strings in an Enum called `Difficulty`. Also the values for each difficulty could be saved in the Constants class.

File: src/main/java/at/ac/fhcampuswien/snake/util/StateManager.java [hunk 3]
```diff
@@ -22,6 +22,8 @@
 public class StateManager {
     private static Stage stage = null;
 
+    public static int refreshTime = 200;
```
Comment (sorinlzr) [inline src/main/java/at/ac/fhcampuswien/snake/util/StateManager.java:25]: If you save the value for the normal difficulty in the Constants class in a variable you can reuse it here also (see comment in the MainViewController class)

**Review by sorinlzr — CHANGES_REQUESTED** [review]

File: src/main/java/at/ac/fhcampuswien/snake/controller/MainViewController.java (hunk 2 above)
Comment (benniii1) [reply src/main/java/at/ac/fhcampuswien/snake/controller/MainViewController.java:66]: We tried but didn't manage to assign values like 300/200/150 to the enums, So we just put it directly to the dropdown mapping.

Comment (benniii1) [reply src/main/java/at/ac/fhcampuswien/snake/controller/MainViewController.java:66]: I looked it up, I mean it is possible, but we would need to define a member variable of the enum, so for example a call would look like HARD.TIME, MEDIUM.TIME,...

**Review by sorinlzr — APPROVED** [review]
lgtm
